src/styles/block/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  justify-content: center;
+  align-items: center;
+
+  &__image {
+    display: flex;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
```

Comment:
The '.card__image' class has 'display: flex;' but it does not seem to have any flex items inside it. If there are no flex items, the 'display: flex;' property is unnecessary.